torchdata/datapipes/iter/transform/callable.py

```diff
@@ -162,6 +165,115 @@ def __len__(self) -> int:
         raise TypeError(f"{type(self).__name__}'s length relies on the output of its function.")
 
 
+# TODO(VitalyFedyunin): Replacing FlatMapperIterDataPipe will require BC breaking change of input_col behaviour
```

Comment:
Is this because when `input_col` is specified`, the new version will try to return a list/tuple if the original input is a list/tuple?